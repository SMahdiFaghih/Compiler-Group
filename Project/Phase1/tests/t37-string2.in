"Aren't you a "string"?"