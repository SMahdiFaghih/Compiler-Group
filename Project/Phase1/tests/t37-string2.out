T_STRINGLITERAL "Aren't you a "
string
T_STRINGLITERAL "?"
